### ISSUE TITLE
[CAP-813] How do I get started on AKS?

### ISSUE THREAD

**Issue by btat** [SUSE/doc-cap#724]
See also: https://github.com/SUSE/kubecf/issues/18

```
az aks create -g RESOURCE_GROUP -n kubecf-aks --node-count 3 --admin-username azuser --ssh-key-value /PATH/TO/id_rsa_.pub --node-vm-size Standard_DS4_v2 --node-osdisk-size=80 --nodepool-name NODEPOOL_NAME --service-principal .... --client-secret ....

k create ns cfo

followed by the 2 helm installs
```

**Comment by troytop** [comment]
When you get to this, check to see if the Blue Horizon installer GUI (in the marketplace) is supporting CAP 2.0. Docs should point to that and the cap-terraform scripts as options to a completely manual installation.

**Comment by btat** [comment]
https://azuremarketplace.microsoft.com/en-us/marketplace/apps/suse.cap-deploy-byos?tab=Overview

**Comment by btat** [comment]
Closed via #764, #815, #838, #840, #846, #851, #859, #861, #863

#883 Created for BH
